README.md

```diff
@@ -50,13 +50,13 @@ Utilizing a variety of smooth built-in animations, modal dialog opening and clos
 [Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-dialog-github-samples)
 
```

Comment:
Use this link for the below section "https://learn.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-7.0"
![image](https://github.com/SyncfusionExamples/Create-a-Dialog-Component-in-a-Blazor-Server-Application/assets/95272441/90fc844d-5ecf-48c2-9d77-946ca6d908fd)

Use this link for the below section : https://support.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=blazor-dialog-github-samples
![image](https://github.com/SyncfusionExamples/Create-a-Dialog-Component-in-a-Blazor-Server-Application/assets/95272441/d41d46ff-1917-48ed-900a-039dc9206ba4)